### PR TITLE
refactor(SectionHeader): update section header component

### PR DESCRIPTION
**Pull request by radiansz**

### CODE REVIEW

File: client/components/SectionHeader/SectionHeader.css [hunk 1]
```diff
@@ -6,9 +6,57 @@
 }
 
 .section-header__container {
+  align-items: center;
+  display: flex;
   height: 100px;
   margin: 0 auto;
   max-width: 1150px;
   padding-bottom: 20px;
```
Comment (rosendi) [inline client/components/SectionHeader/SectionHeader.css]: Remove it.

File: client/components/SectionHeader/SectionHeader.js [hunk 2]
```diff
@@ -1,19 +1,45 @@
...
     <div {...element('container')}>
-      {children}
+      {backlink && (
+        <Link {...element('backlink')} to={backlink.to}>
+          <ArrowLeftIcon {...element('arrow')} />
+          {backlink.label}
+        </Link>
+      )}
+
+      <div {...element('title')}>
```
Comment (rosendi) [inline client/components/SectionHeader/SectionHeader.js]: `<h1 />`

File: client/routes/subscribers/components/SubscribersTitle.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  },
+});
+
+const SubscribersTitle = ({ count, block, element, loading }) => {
+  if (loading) {
+    return <Spinner />;
+  }
+
+  return (
+    <div {...block()}>
```
Comment (rosendi) [inline client/routes/subscribers/components/SubscribersTitle.js]: Just use span and don't use styles.

Comment (radiansz) [inline client/routes/subscribers/components/SubscribersTitle.js]: @rosendi amount of subscribers is a link and styled [here](https://app.sendsay.ru/subscribers/lists). Are we ignoring it?

**Comment by zouxuoz** [comment]
lgtm

**Comment by rosendi** [comment]
lgtm

**Pull request merged**